app/scripts/inpage.js

```diff
@@ -36,6 +39,10 @@ import { WindowPostMessageStream } from '@metamask/post-message-stream';
 import { initializeProvider } from '@metamask/providers/dist/initializeInpageProvider';
 import shouldInjectProvider from '../../shared/modules/provider-injection';
 
+// provider constants
+const iconUrl =
+  'https://raw.githubusercontent.com/MetaMask/brand-resources/cb6fd847f3a9cc5e231c749383c3898935e62eab/SVG/metamask-fox.svg'; // TODO: Find a shorter URL
```

Comment:
Do we have another url we could use?

Maybe `browser.runtime. getURL("asset_name.svg")` paired with `web_accessible_resources`?

Reply:
I'd recommend that we use an SVG data URI instead. That's the only way to avoid forcing the dapp to make a network call to use the image.

If it were our code, we'd use an image local to the extension. But sites aren't able to reference files that are part of an extension, browsers won't allow it.

But SVGs can be reasonably small as data URIs.

Reply:
It may be possible to expose resources from extension? https://developer.chrome.com/docs/extensions/mv3/manifest/web_accessible_resources/

Reply:
We've run into security issues in the past using those. We can consider it but we'd have to review it very carefully, there are a few tricky was to use those to bypass security restrictions

Reply:
FYI I did update the spec to [make this a URI only](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-6963.md#imagesicons) now for the reasons @Gudahtt points out.